### DISCUSSION TITLE
Recreating higher quality previews

### DISCUSSION THREAD

**Discussion by rahul003** [photoprism/photoprism#1412] [Q&A] [answered]
Is reindexing all files enough to generate new previews of higher quality (After changing settings for static file size and the method for creating previews)? Or do I have to clean up something and generate them again?

**Comment by lastzero** [comment, marked as answer]
Yes. To recreate all thumbs after changing quality settings, you have to delete all JPEGs in the `storage/cache` folder and then run `photoprism thumbs` in a terminal (fastest) or reindex so that new thumbs will be pre-rendered.
